app/views/partials/_nav.html.erb

```diff
@@ -0,0 +1,61 @@
+<nav class="navbar" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+    <a class="navbar-item" href="#">
+      <p>SocietalBook</p>
+    </a>
+
+    <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+    </a>
+  </div>
+
+  <div id="navbarBasicExample" class="navbar-menu">
+    <div class="navbar-start">
+
+      <%= link_to 'Home', root_path, class: "navbar-item"%>
+
+      <%= link_to 'Micropost', new_microposts_path, class: "navbar-item"%>
+
```

Comment:
**[OPTIONAL]**
Add a link for micro-posts index, or make the main page to display all micro-posts.